azure-pipelines.yml

```diff
@@ -110,4 +110,13 @@ jobs:
         PathtoPublish: '$(Build.SourcesDirectory)/artifacts/log/$(_configuration)'
         ArtifactName: 'Windows $(_configuration)'
       continueOnError: true
-      condition: always()
\ No newline at end of file
+      condition: always()
+
+- job: Markdownlint
+  pool:
+      vmImage: ubuntu-latest
+  steps:
+    - script: sudo npm install -g markdownlint-cli
+      displayName: Install markdownlint-cli
+    - script: markdownlint '**/*.md' -i '**/AnalyzerReleases.*.md' -i 'docs/Analyzer reference page template.md' -i 'eng/common/**/*.md'
```

Comment:
The `-i` switch used here is "ignore". The currently ignored files have some violations. We probably want to fix them and enable lint for these files in the future.

Reply:
Do we have a ticket tracking this remaining task?